test/e2e/step-definitions/journal-steps.ts

```diff
@@ -108,31 +134,31 @@ Then('I should have a category {string} test for {string}', (category, candidate
 });
 
 Then('The vehicle for {string} has length {string}, width {string}, height {string} and seats {string}',
-(candidateName, length, width, height, seats) => {
-  const lengthValue = getElement(
-    by.xpath(`//vehicle-details/div/span/span[text()= 'L: ']/following-sibling::span
+  (candidateName, length, width, height, seats) => {
+    const lengthValue = getElement(
+      by.xpath(`//vehicle-details/div/span/span[text()= 'L: ']/following-sibling::span
     [ancestor::ion-grid/ion-row/ion-col/ion-grid/ion-row/ion-col/candidate-link/div/button/span/h3
       [text() = "${candidateName}"]]`));
-  expect(lengthValue.getText()).to.eventually.equal(length);
+    expect(lengthValue.getText()).to.eventually.equal(length);
 
-  const widthValue = getElement(
-    by.xpath(`//vehicle-details/div/span/span[text()= 'W: ']/following-sibling::span
+    const widthValue = getElement(
+      by.xpath(`//vehicle-details/div/span/span[text()= 'W: ']/following-sibling::span
     [ancestor::ion-grid/ion-row/ion-col/ion-grid/ion-row/ion-col/candidate-link/div/button/span/h3
       [text() = "${candidateName}"]]`));
-  expect(widthValue.getText()).to.eventually.equal(width);
+    expect(widthValue.getText()).to.eventually.equal(width);
 
-  const heightValue = getElement(
-    by.xpath(`//vehicle-details/div/span/span[text()= 'H: ']/following-sibling::span
+    const heightValue = getElement(
+      by.xpath(`//vehicle-details/div/span/span[text()= 'H: ']/following-sibling::span
     [ancestor::ion-grid/ion-row/ion-col/ion-grid/ion-row/ion-col/candidate-link/div/button/span/h3
       [text() = "${candidateName}"]]`));
-  expect(heightValue.getText()).to.eventually.equal(height);
+    expect(heightValue.getText()).to.eventually.equal(height);
 
-  const seatValue = getElement(
-    by.xpath(`//vehicle-details/div/span/span[text() = 'Seats: ']/following-sibling::span
+    const seatValue = getElement(
+      by.xpath(`//vehicle-details/div/span/span[text() = 'Seats: ']/following-sibling::span
     [ancestor::ion-grid/ion-row/ion-col/ion-grid/ion-row/ion-col/candidate-link/div/button/span/h3
       [text() = "${candidateName}"]]`));
-  return expect(seatValue.getText()).to.eventually.equal(seats);
-});
+    return expect(seatValue.getText()).to.eventually.equal(seats);
```

Comment:
can you create a helper function for the expect assertion

Reply:
Done.